llama/runner/README.md

```diff
@@ -19,3 +19,39 @@ curl -X POST -H "Content-Type: application/json" -d '{"prompt": "hi"}' http://lo
 ```
 curl -X POST -H "Content-Type: application/json" -d '{"prompt": "turn me into an embedding"}' http://localhost:8080/embeddings
 ```
+
+## Building
+
+To build the `runner` executable, you first need to build the parent `llama` module:
+
+```sh
+(cd .. && go build .)
+```
+
+From there, you can build this executable:
+
+```sh
+go build .
+```
```

Comment:
In the general case, you need to use `make` to build for CUDA, etc. so these instructions aren't quite right. I think rather than repeating directions here (they can get somewhat complex on different platforms), we can just leave it at what is in the top level README. That will build the runners as appropriate.

Reply:
Good catch. I added these steps as notes for myself as I was trying to figure out the build process. I'll pull them back out.